frontend/src/lib/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -215,7 +215,9 @@ class Multiselect extends React.PureComponent<Props, State> {
     const { options } = element
     const disabled = options.length === 0 ? true : this.props.disabled
     const placeholder =
-      options.length === 0 ? "No options to select." : "Choose an option"
+      options.length === 0
+        ? this.props.element.placeholderNoOptions
+        : this.props.element.placeholder
```

Comment:
nit: would recommend destructuring `placeholderNoOptions` and `placeholder` from `element` on line 215 above just so that this code is a bit less verbos

Reply:
Thank you for your suggestion. Fixed in the following commit.
https://github.com/streamlit/streamlit/pull/6901/commits/216c9f1dadf5060629778a356bcf29df131e4ec2